### DISCUSSION TITLE
Welcome to satellite-image-deep-learning Discussions!

### DISCUSSION THREAD

**Discussion by robmarkcole** [satellite-image-deep-learning/techniques#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bkermanlaraonca** [comment]
Robin: I have been studying radar (and 1 satellite) images of a hurricane to derive its change in strength/intensity with time based on the properties of the images. The strengthening of a hurricane involves both changes in rotational energy and in its convectivity (rain rate). I have found that clusters can be assembled by sorting for annealing, and then applying entropic force (see WIKI) definition to isolate rotation in each cluster. The ratio of the ratio of image integrated rotation to terminal velocity to the ratio of (convectivety) -  latent heating to dissipation rates produces an index which varies with the hurricanes energy as measured by aircraft sorties through the hurricane.

The key variable that any analysis seeking to estimate a hurricane's strength needs to be able to characterize the rotational and convective aspects implied in the imagery.

I tried the analysis on satellite imagery (GOES) but it does not provide as strong 'signal' output because it represents the vertical integration of the returned IR.

I'm a 'Cambridge' product of sorts also as I did a sabbatical at DAMTP (in the dark ages ca1990). There is a trail from some co-operation with Scott Polar which involved 1st year Arctic ice rafting especially its connectedness in other imagery. 

Pleased to be asked any questions.

Bryan Kerman

**Reply by robmarkcole** [reply]
Hi Bryan, great to hear from you. Based on what you have said I think you would have a lot to discuss with Eduardo who I interviewed in https://www.satellite-image-deep-learning.com/p/worldsphereai-with-eduardo-siman

**Comment by MuhammedM294** [comment]
Hey Robin! Thank you for the considerable effort you're making to manage this repo. It's fantastically overwhelming. I wonder why you didn't number each technique and its sub-section. I think it'll be more navigable this way.

**Reply by robmarkcole** [reply]
That's a good suggestion :) Could you make a PR to implement it? Thanks

**Reply by MuhammedM294** [reply]
Thanks. I'll make it today.
